src/main/java/core/basesyntax/converter/DataConverterImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.converter;
+
+import core.basesyntax.transaction.FruitTransaction;
+import java.util.ArrayList;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    @Override
+    public List<FruitTransaction> converterToTransaction(List<String> inputData) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (String data: inputData) {
+            if (data.equals("type,fruit,quantity")) {
+                continue;
+            }
+            String[] parts = data.split(",");
+            String operationCode = parts[0].trim();
+            String fruitName = parts[1].trim();
+            int quantity = Integer.parseInt(parts[2].trim());
+
+            // Determine the type of activity
+            FruitTransaction.Operation operation
+                    = FruitTransaction.Operation.fromCode(operationCode);

```

Comment:
This comment is not providing any additional information that is not already clear from the code. It's better to remove it. Remember, comments should explain why the code is doing something, not what it's doing. Good code should be self-explanatory.